Automattic/wp-calypso
# Recurring Payments: Migrate Image to Webpack

#### Changes proposed in this Pull Request

This migrates the one usage of the Recurring Payments to Webpack. For details on the reasoning behind this, see #36850.

It also minimises the file, which seems to be the pattern with the rest of the images in this file (still the same image!)

#### Testing instructions

There should only be one use of this image: https://github.com/Automattic/wp-calypso/search?q=illustration-nomemberships.svg&unscoped_q=illustration-nomemberships.svg

You can find it by visiting `/me/purchases/other` on an account with no recurring payments. Verify that the image is still there and no different. cc @artpi 

<img width="833" alt="Screenshot 2019-11-28 at 18 02 22" src="https://user-images.githubusercontent.com/43215253/69826020-43d10980-1209-11ea-9c0e-3df568540f81.png">
